README.md

```diff
@@ -58,14 +58,54 @@ for idx, item in enumerate(results['items']):
     print(idx, track['artists'][0]['name'], " – ", track['name'])
 ```
 
-## Reporting Issues
+## FAQ and Suggestions
 
 For common questions please check our [FAQ](FAQ.md).
 
 You can ask questions about Spotipy on
 [Stack Overflow](http://stackoverflow.com/questions/ask).
 Don’t forget to add the *Spotipy* tag, and any other relevant tags as well, before posting.
 
-If you have suggestions, bugs or other issues specific to this library,
+If you have suggestions or other issues specific to this library,
 file them [here](https://github.com/plamere/spotipy/issues).
-Or just send a pull request.
+Or just send a pull request. 
+
+Refer to Documentation below for collecting information for bug reporting.
+
+## Bud Reporting
```

Comment:
This doesn't need to be in the README. Please create a PR to update this file directly if you think anything is missing in there .github/ISSUE_TEMPLATE/bug_report.md